src/shapes/Text/Text.ts

```diff
@@ -1816,10 +1801,18 @@ export class Text<
     textContent = textContent
       .replace(/^\s+|\s+$|\n+/g, '')
       .replace(/\s+/g, ' ');
```

Comment:
can the remove svg spaces util be used here?

Reply:
no as said before ths function is out of scope.